thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/AnomalyFlattenResource.java

```diff
@@ -180,6 +175,7 @@ public AnomalyFlattenResource(MergedAnomalyResultManager mergedAnomalyResultDAO,
         resultMap.put(key, dimensionMap.get(key));
       }
       for (String metric : metrics) {
+        resultMap.put(metric, metricValues.get(dimensionMap).getOrDefault(metric, Double.NaN));
```

Comment:
Looks like the line is copied here. Maybe consider
`resultMap.put(metric, metricValues.getOrDefault(dimensionMap, Collections.emptyMap()).getOrDefault(metric, Double.NaN));`

Reply:
Looks good.